### ISSUE TITLE
[Firefox] slider moves to 0 when moved by the user. 

### ISSUE THREAD

**Issue by pshihn** [wiredjs/wired-slider#3]
Related to absolute position of the slider. May not be 0 based on that.

**Comment by smougel** [comment]
I've already solve that in the past on a project with draggables and droppables elements.
Take the element and loop over the parents... Make a sum of offsetHeight and offsetWidth...

Pseudo code : 
```
x=0;
y=0;
while(elem) {
 y+=elem.offsetHeight;
 x+=elem.offsetWidth;
 
    if(elem.hasParent()) {
        elem = elem.parent();
    } else {
        elem = false;
    }
}
```
x and y should have correct coordinate
I'll try to find a working example...

**Comment by pshihn** [comment]
Fixed in v0.5.5
https://github.com/wiredjs/wired-slider/commit/13fd11eb3203c20f5e5dd460df629cd7c93709bd
